### PR TITLE
REF Deduplicate recaptcha handling code

### PR DESCRIPTION
**Pull request by mattwire**
Overview
----------------------------------------
Same code existed on about 11 forms to add the recaptcha.

Before
----------------------------------------
Code to add recaptcha to form in multiple places.

After
----------------------------------------
Single shared function to add recaptcha.

Technical Details
----------------------------------------
Move function from contribution form to CRM_Utils_Recaptcha and call from all forms.
Also remove singleton where it is not required.

Comments
----------------------------------------

### CODE REVIEW

File: CRM/Utils/ReCAPTCHA.php [hunk 1]
```diff
@@ -120,4 +120,15 @@ public static function add(&$form) {
     }
   }
 
+  /**
+   * Enable ReCAPTCHA on Contribution form
+   */
+  public static function enableCaptchaOnForm(&$form) {
```
Comment (pradpnayak) [inline CRM/Utils/ReCAPTCHA.php]: Need to document $form

**Comment by pradpnayak** [comment]
Looks good to me, Nice cleanup.

File: CRM/Utils/ReCAPTCHA.php (hunk 1 above)
Comment (mattwire) [reply CRM/Utils/ReCAPTCHA.php]: Done thanks :-)

**Comment by monishdeb** [comment]
It's a straight forward change and a nice cleanup indeed :) 


Did a quick test on Signsture and online event reg form and doesn't break usual behavior. Happy with the patch. Merging now. 

Thanks @mattwire @pradpnayak for making this PR mergeable.

**Pull request merged**